lib/broccoli/angular2-app.js

```diff
@@ -87,10 +87,16 @@ Angular2App.prototype.toTree = function () {
     destDir: 'vendor'
   });
 
+  var publicDirTree = new Funnel(path.join(sourceDir, 'public'), {
+    destDir: 'public',
```

Comment:
The `public` directory should go to the root directory, so in this case you can remove `destDir`.